tests/components/bsblan/conftest.py

```diff
@@ -53,9 +53,13 @@ def mock_bsblan() -> Generator[MagicMock]:
         )
         bsblan.state.return_value = State.from_json(load_fixture("state.json", DOMAIN))
 
-        bsblan.static_values.return_value = StaticState.from_json(
-            load_fixture("static.json", DOMAIN)
-        )
+        # Patch static values based on the test case
+        async def set_static_values(param):
+            bsblan.static_values.return_value = StaticState.from_json(
+                load_fixture(param, DOMAIN)
+            )
+
+        bsblan.set_static_values = set_static_values
```

Comment:
Let's not do this here, we should make the mock be like the real life object ideally

Reply:
Why don't we just set the static_values to a good default and then just overwrite in the test when we need to?